sandbox/experimental/data-packages/dwc-dp/0.1/index.json

```diff
@@ -1,1013 +1,1013 @@
 {
   "identifier": "http://rs.tdwg.org/dwc/dwc-dp",
-  "url": "https://rs.gbif.org/sandbox/experimental/data-packages/dwc-dp/0.1/index.json",
+  "url": "",
   "name": "dwc-dp",
   "version": "0.1",
   "title": "Darwin Core Data Package",
   "shortTitle": "dwc-dp",
   "description": "A data package for sharing biodiversity data using Darwin Core.",
-  "issued": "2025-09-17",
+  "issued": "2025-09-26",
   "isLatest": true,
   "tableSchemas": [
     {
-      "identifier": "http://rs.tdwg.org/dwc/dwc-dp/table-schemas/agent",
-      "url": "https://rs.gbif.org/sandbox/experimental/data-packages/dwc-dp/0.1/table-schemas/agent.json",
```

Comment:
@mike-podolskiy90 do you also need full URLs in the index.json?

Reply:
Yes, we do